### PR TITLE
Pull out polytone

### PR DESCRIPTION
**Pull request by Buckram123**
This PR aims to pull out used types for polytone from polytone and polytone_note packages.
### Checklist

- [ ] CI is green.
- [ ] Changelog updated.

### CODE REVIEW

File: framework/packages/abstract-std/src/native/ibc.rs [hunk 1]
```diff
@@ -12,6 +13,7 @@ use crate::{
     ibc_client,
     objects::{module::ModuleInfo, TruncatedChainId},
 };
+use polytone_callbacks::{Callback as PolytoneCallback, ErrorResponse, ExecutionResponse};
```
Comment (Kayanski) [inline framework/packages/abstract-std/src/native/ibc.rs:16]: Whats'up here ?

**Comment by Kayanski** [comment]
Very nice, could you add references to the code copied (with URLs for instance), because we might/will touch on this later and for reference ?

File: framework/packages/abstract-std/src/native/ibc.rs (hunk 1 above)
Comment (Buckram123) [reply framework/packages/abstract-std/src/native/ibc.rs:16]: What do you mean?

**Comment by Buckram123** [comment]
> Very nice, could you add references to the code copied (with URLs for instance), because we might/will touch on this later and for reference ?

Added links to docs.rs of copied types

**Review by CyberHoward — APPROVED** [review]

**Pull request merged**